HL7-DaVinci/CDS-Library
# DMEERX-829 Using SDC InitialExpression instead of cqf-experssion

This PR uses SDC initialExpression to prepopulate the questionnaire instead of cqf-expression.

To test:
Using the 829-HOT-SDC branch in DTR repo and verify the prepopulation of HomeOxygenTherapy order form works. 

Note and question:
@lpdecelles in this story: https://jira.mitre.org/browse/DMEERX-812 We said to keep HOT as cqf-expression. Shall we pick another topic instead of HOT to keep cqf-expression?
